### PR TITLE
[SDCI] Clarify documentation on tags and measures

### PR DESCRIPTION
**Pull request by CarlosNietoP**
### What does this PR do? What is the motivation?

Fixing unclear information on how to add more than one tag and measure

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by edrevo — APPROVED** [review]

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 1]
```diff
@@ -76,15 +76,15 @@ Tags can be added to the pipeline span or to the job span.
...
 
 ```shell
-DATADOG_SITE={{< region-param key="dd_site" >}} datadog-ci tag [--level <pipeline|job>] [--tags <tags>]
+DATADOG_SITE={{< region-param key="dd_site" >}} datadog-ci tag [--level <pipeline|job>] [--tags <tag1>] [--tags <tag2>] ...
 ```
 
 You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY` and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
 
-The following example adds the tag `team` to the pipeline span.
+The following example adds the tags `team` and `service` to the pipeline span.
```
Comment (neko-dd) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:84]:
```suggestion
The following example adds `team` and `service` tags to the pipeline span.
```

File: content/en/continuous_integration/pipelines/custom_tags_and_measures.md [hunk 2]
```diff
@@ -102,15 +102,15 @@ To create a facet from a tag, click the gear icon next to a tag name on the [Pip
...
 
 ```shell
-DATADOG_SITE={{< region-param key="dd_site" >}} datadog-ci measure [--level <pipeline|job>] [--measures <measures>]
+DATADOG_SITE={{< region-param key="dd_site" >}} datadog-ci measure [--level <pipeline|job>] [--measures <measure1>] [--measures <measure2>]...
 ```
 
 You must specify a valid [Datadog API key][3] using the environment variable `DATADOG_API_KEY` and the [Datadog site][12] using the environment variable `DATADOG_SITE`.
 
-The following example adds the measure `error_rate` to the pipeline span:
+The following example adds the measures `error_rate` and `size` to the pipeline span:
```
Comment (neko-dd) [inline content/en/continuous_integration/pipelines/custom_tags_and_measures.md:110]:
```suggestion
The following example adds `error_rate` and `size` measures to the pipeline span:
```

**Review by neko-dd — APPROVED** [review]
Two suggestions but overall LGTM

**Review by neko-dd — APPROVED** [review]

**Pull request merged**